src/game/common/damagefx.h

```diff
@@ -31,6 +32,25 @@ class DamageFX
 public:
     DamageFX() {}
 
+    void Clear()
+    {
+        for (int i = 0; i < DAMAGE_NUM_TYPES; i++) {
+            for (int j = 0; j < VETERANCY_COUNT; j++) {
```

Comment:
Perhaps use `dmg`, `vet` names for damage and veterancy. Several times in this function.